caffe2/utils/string_utils.h

```diff
@@ -10,7 +10,8 @@
 
 namespace caffe2 {
 
-CAFFE2_API std::vector<std::string> split(char separator, const std::string& string);
+CAFFE2_API std::vector<std::string>
+split(char separator, const std::string& string, bool ignore_empty = false);
```

Comment:
is this change necessary? I don't see a use of the third variable in the code

Reply:
That was for the glow transform, it originally relied on this but it was gone via rebase. I think they want to recover it back.